deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
Thank you for asking!   

This is the end of the output which shows the failure.
http://pastebin.test.redhat.com/412553

It seems it fails at the line 234 in "hack/testing/test-upgrade.sh".
234       [[ -z "$(oc logs $UPGRADE_POD | grep 'Migration for project '$project': {"acknowledged    ":true}')" ]] && return 1